### PR TITLE
add query for counting databases in a cluster

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1224.surge.sh

File: modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc [hunk 1]
```diff
@@ -319,6 +319,18 @@ RETURN count(*) AS count
 +-------+
 ----
 
+In a cluster, the `SHOW DATABASES` returns the number of allocations of databases and may be misleading if the number of distinct databases is desired.
+In this case, the number of distinct databases hosted in a cluster can be seen using a `count()` aggregation with `YIELD` and `RETURN` with `name` instead of the wildcard.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc:323]:
```suggestion
In this case, the `count()` function can be used in the `RETURN` clause to aggregate the number of `DISTINCT` database names: 
```

**Review by renetapopova — APPROVED** [review]
I have a small suggestion. Otherwise, it looks good to me. Thanks, @AlexicaWright!

File: modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc [hunk 2]
```diff
@@ -300,13 +300,13 @@ SHOW DATABASE movies YIELD *
 
 === Show the number of databases
 
-The number of databases can be seen using a `count()` aggregation with `YIELD` and `RETURN`.
+The number of distinct databases can be seen using `YIELD` and a `count()` function in the `RETURN` clause to aggregate the number of `DISTINCT` database names.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc:303]:
```suggestion
The number of distinct databases can be seen using `YIELD` and a `count()` function in the `RETURN` clause:
```

**Pull request merged**